SPECS-EXTENDED/gi-docgen/gi-docgen.signatures.json

```diff
@@ -0,0 +1,6 @@
+{
+ "Signatures": {
+  "gi-docgen-2024.1.tar.bz2": "f4420b3be6e61584066655170cf443312fe101907c02b44650ef4b206ca2c4f5",
+  "gi-docgen-2022.2-no-web-fonts.patch": "3dd950ffd42904ddec50b45a060cdd1d9d2b56eba02c2f9f78df1c9445c247da"
```

Comment:
is patch file needed in signatures file?

Reply:
IIRC, yes it is.

Reply:
Got the clarification from the build team. Everything mentioned under the spec's "Sources" section must be signed in signatures.json. That said, patches are not captured as part of "Sources" section, they are captured as part of the "Patch" section. So, they don't need to be signed.

In short. Let me fix this.

Reply:
Fixed.